### DISCUSSION TITLE
✅ Sync/export settings

### DISCUSSION THREAD

**Discussion by mbierman** [isontheline/pro.webssh.net#143] [Ideas]
It would be awesome if I could share or sync server settings. The logins, saved commands, etc. even if it is just airdropping...

**Comment by isontheline** [comment]
Hello Michael 👋

Yes I have planned the sync feature [here](https://github.com/isontheline/pro.webssh.net/discussions/43)

**Reply by mbierman** [reply]
Excellent. Feel free to close as duplicate. And thank you.

**Comment by isontheline** [comment]
Also related to this discussion : https://github.com/isontheline/pro.webssh.net/discussions/132

**Comment by isontheline** [comment]
iCloud is now supported since WebSSH 14.15

**Comment by isontheline** [comment]
You can also backup and restore database : https://webssh.net/documentation/help/howtos/mashREPL/database-backup/
